callosum/message.py

```diff
@@ -122,7 +126,7 @@ def failure(cls, request, exc):
             MessageTypes.FAILURE,
             request.method, request.order_key, request.seq_id,
             ErrorMetadata(type(exc).__name__, ''),  # TODO: format stack
-            mpackb(tuple(map(str, exc.args))),
+            Message.mpackb(tuple(map(str, exc.args))),
```

Comment:
Use `cls` here and other instances of `Message.mpackb()` and `Messge.munpackb()`.